packages/graphql-generator/package.json

```diff
@@ -17,18 +17,23 @@
     "aws"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && yarn check-codegen-version",
```

Comment:
Including the version check in the build process. As discussed, this will push the developer to look at the maintenance instructions if the version from npm is different than the one adapted in the vendor folder.